pages/data-providers/[data-provider-id]/compliance.css

```diff
@@ -15,9 +15,18 @@
   }
 }
 
-.compliance-level {
+.data-overview {
   flex: 2 2 60%;
   width: 100%;
+
+  & > .numbers {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: flex-end;
```

Comment:
I am not sure we should align numbers to the bottom; `baseline` or `flex-start` should be fine. What do you try to cover by this line?